SRC/GAME.CPP

```diff
@@ -23,7 +23,8 @@ int ipxofs[MAX_PLAYERS];
 char ipxstream[MAX_PLAYERS][MAXDATASIZE];
 struct nodeaddr serveraddr;
 char servername[10];
-struct packet *send, *rec[RECEIVERS];
+// send defined here overrides send() and bad things happen
+//struct packet *send, *rec[RECEIVERS];

```

Comment:
Is it enough to make `send` `static`, since it's only used in this file?

Reply:
Hmm probably. But I guess it will eventually be deleted anyway, or are we going to build an IPX compatibility layer? :thinking:

Reply:
Not sure if we want to delete the IPX code, maybe we'll add multiplayer support in the future?

But I guess those variables can be commented out because the code using them is also commented out.